src/clientManager.js

```diff
@@ -93,6 +89,50 @@ ClientManager.prototype.addUser = function(name, password) {
 	return true;
 };
 
+ClientManager.prototype.updateUser = function(name, opts) {
+	var users = this.getUsers();
+	if (users.indexOf(name) === -1) {
+		return false;
+	}
+	if (typeof opts === "undefined") {
+		return false;
+	}
+	var path = Helper.HOME + "/users/" + name + ".json";
+	var user = {};
+
+	try {
+		user = this.readUserConfig(name);
+		_.merge(user, opts);
+	} catch (e) {
+		console.log(e);
+		return;
+	}
+
+	fs.writeFileSync(
+		path,
+		JSON.stringify(user, null, " "),
+		{mode: "0777"},
```

Comment:
We should have `0600` or at least `0644` here, but that's a thing to fix in a separate PR.